### PR TITLE
26 local sandbox

### LINKED ISSUE #26: Local sandbox command
Child of #16

I want to pull a site database and run deployment commands, so that I can have a local sandbox that mirrors production.

In ahoy, this is accomplished with the site up command. 

Acceptance criteria:
- Project configuration tells command where to find database
- imports database
- download the files

### PR DESCRIPTION
**Pull request by fmizzell**
Fixes #26

`dktl dkan:restore`

Also adds ability to call dktl within CLI container, so that we can for instance call `_exec('dktl whatever')` in a command and it will work.

## QA Steps

1. New folder, `dktl dc up -d`
2. `dktl init`
3. `dktl dkan:get 7.x-1.15.2 && dktl dkan:make && dktl drupal:make`
3. `dktl dkan:restore https://s3.amazonaws.com/dkan-tools/last_install.sql https://s3.amazonaws.com/dkan-tools/files.zip`

### CODE REVIEW

File: bin/dktl.sh [hunk 1]
```diff
@@ -76,7 +76,14 @@ else
     if [ -z "$VENDOR" ]; then
         $BASE_DOCKER_COMPOSE_COMMAND exec cli composer install --working-dir=/usr/local/dkan-tools/
     fi
-    $BASE_DOCKER_COMPOSE_COMMAND exec cli php /usr/local/dkan-tools/bin/app.php $1 "${@:2}"
+
```
Comment (dafeder) [inline bin/dktl.sh]: Add some comments so it's clearer what's actually happening here

File: bin/dktl.sh [hunk 2]
```diff
@@ -76,7 +76,14 @@ else
     if [ -z "$VENDOR" ]; then
```
Comment (dafeder) [inline bin/dktl.sh]: Check this logic, seems to always run composer install

Comment (fmizzell) [reply bin/dktl.sh]: I saw this when we were working together, but I was not able to replicate. When the vendor directory is present, composer install does not run.

**Comment by dafeder** [comment]
Waiting on changes to #17 and rebase to review

**Pull request closed without merging**